components/org.wso2.carbon.identity.conditional.auth.functions.http/src/test/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImplTest.java

```diff
@@ -162,7 +162,8 @@ public void testHttpGetWithNullHeaders() {
     @Test(expectedExceptions = IllegalArgumentException.class)
     public void testHttpGetWithInvalidNumberOfArguments() {
         Map<String, Object> eventHandlers = new HashMap<>();
-        httpGetFunction.httpGet(getRequestUrl("dummy-get"), eventHandlers, eventHandlers, eventHandlers);
+        httpGetFunction.httpGet(getRequestUrl("dummy-get"),
+                eventHandlers, eventHandlers, eventHandlers, eventHandlers);
```

Comment:
These unit functions should be able to test a mocked request

Reply:
Will add a unit test for this improvement

Reply:
Fixed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/afe4342c547dfe83345abfd23f7fee114389532b